R/aaa.R

```diff
@@ -33,97 +39,114 @@ dots_env_copy <- function(dots) {
 
 # Given a function and dots, rename dots based on how things will positionally match
 #' @importFrom rlang is_empty
-rename_dots <- function(handler, dots, addData=TRUE){
+rename_dots <- function(handler, dots, addData = TRUE) {
   if (is_empty(dots)) {
     return(dots)
   }
-  
+
   f <- function(...) as.list(match.call(handler)[-1])
-  
+
   d_idx <- setNames(seq_along(dots), names(dots))
-  
+
   if (addData) {
     d_idx["data"] <- list(NULL)
   }
-  
+
   d_idx <- eval_tidy(quo(f(!!!d_idx)))
-  
+
   if (addData) {
     d_idx$data <- NULL
   }
-  
+
   d_idx <- unlist(d_idx)
-  
+
   d_idx <- d_idx[names(d_idx) != "" ]
-  
+
   names(dots)[d_idx] <- names(d_idx)
-  
+
   dots
 }
 
 # Returns a new function(data) which calls FUN(data, dots)
-currydata <- function(FUN, dots, addDataArg = TRUE, strictDataParam = TRUE, cloneDots = TRUE) {
-  # heuristic to reuse deep clones
+currydata <-
+  function(FUN,
+           dots,
+           addDataArg = TRUE,
+           strictDataParam = TRUE,
+           cloneDots = TRUE) {
```

Comment:
This is bad style in general, if this was automated, please change the setting and keep things on one line if less than approx 100 chars, as in the R source code..